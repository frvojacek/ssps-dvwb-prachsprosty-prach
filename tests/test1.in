3 3
090
113
023
0