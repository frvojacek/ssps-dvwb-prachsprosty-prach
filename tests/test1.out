080
002
012

070
001
001

060
000
000

050
000
000

040
000
000

030
000
000

020
000
000

010
000
000

000
000
000
